packages/dso-toolkit/src/components/anchor/anchor.mixins.scss

```diff
@@ -83,3 +90,57 @@
 
   @include di.variant("external-link");
 }
+
+@mixin mode-phone($base-styling: true) {
```

Comment:
De download en extern mixins hebben `$base-styling: false`, dan is het verwarrend dat phone en email het tegenovergestelde gedrag (`$base-styling: true`) hebben.

Door alle mixins uit deze file dezelfde signature te geven neem je de verwarring weg.

De aanroep van download&extern en phone&email in `anchor.scss` is nu ook anders. Voor download en extern wordt eerst expliciet de base mixin aangeroepen. Voor phone en email wordt dat impliciet in de mixins gedaan.

Ik kan mij er ook in vinden als we juiste voor de impliciete variant gaan. De download en extern mixins worden maar één keer gebruikt, dus dat is een minimale change. It's up to you.

Reply:
Ik heb deze branch halverwege van Anneke overgenomen, dus bepaalde beslissingen waren al genomen. even een reset dus ;)

Reply:
@tfrijsewijk heb de boel qua mixins wat slimmer gemaakt, build loopt :)